assembly/pom.xml

```diff
@@ -5,8 +5,8 @@
   <groupId>com.hortonworks.spark</groupId>
   <artifactId>spark-llap-assembly_2.11</artifactId>
   <packaging>pom</packaging>
-  <version>1.1.1-2.1</version>
-  <name>spark-llap-assembly_2.11</name>
+  <version>1.4.1-2.2</version>
```

Comment:
Please fix this, too.
`1.4.1-2.2 -> 1.1.4-2.1`.
Also, it would be better to have `-SNAPSHOT` postfix, too.